config/ldp/metadata-writer/default.json

```diff
@@ -17,6 +18,7 @@
       "handlers": [
         { "@id": "urn:solid-server:default:MetadataWriter_AllowAccept" },
         { "@id": "urn:solid-server:default:MetadataWriter_Constant" },
+        { "@id": "urn:solid-server:default:MetadataWriter_ContentType" },
         { "@id": "urn:solid-server:default:MetadataWriter_Mapped" },
```

Comment:
Content-type will have to be removed from this metadatawriter otherwise behaviour is going to be undefined. This is also what makes this change breaking, since if someone has a custom `ldp/metadata-writer/default.json` config implementation they would suddenly no longer get the content-type response header.